src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -1226,6 +1226,10 @@ public function release_bookmark( $bookmark_name ) {
 	/**
 	 * Moves the internal cursor in the HTML Processor to a given bookmark's location.
 	 *
+	 * Be careful! Seeking backwards to a previous location resets the parser to the
+	 * start of the document and reparses the entire contents up until it finds the
+	 * sought-after bookmarked location. This currently need further optimization.
```

Comment:
I'd drop the bit about it needing optimization. We know there are potential optimizations but we don't need to write that explicitly here.

```suggestion
	 * sought-after bookmarked location.
```

Reply:
Removed this, thank you.